packages/lib/CalEventParser.ts

```diff
@@ -185,10 +185,10 @@ export const getRichDescription = (
   const t = t_ ?? calEvent.organizer.language.translate;
 
   return `
-${getCancellationReason(calEvent, t)}
-${getWhat(calEvent, t)}
-${getWhen(calEvent, t)}
-${getWho(calEvent, t)}
+${getCancellationReason(calEvent, t)}\n
```

Comment:
@snoseeds  why did you add the \n s here it already is returned at the end of each of the functions.